script.js

```diff
@@ -1,4 +1,385 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var username = '';
+var playerSPS = '';
+var compSPS = '';
+var myOutputValue ='';
+var newGameMsg = 'Please enter "scissors", "paper" or "stone" to start a new game.';
+var drawMsg = 'It is a draw, please enter "scissors", "paper" or "stone" to play again.'
+var winMsg = 'You win the game!';
+var loseMsg = 'You lose the game... please enter "scissors", "paper" or "stone" to play again.';
+
+
+var main = function (input){
+if (!playerSPS){
+  if (!input){
+  myOutputValue = newGameMsg;
   return myOutputValue;
-};
+  }
+}
+
+playerSPS = input.toLowerCase();
+compSPS = randomSPS();
+console.log (compSPS);
+
+
+if (playerSPS != 'scissors' &&
+    playerSPS != 'paper' &&
+    playerSPS != 'stone'){
+      myOutputValue = newGameMsg;
+    } 
+
+if (playerSPS == compSPS){  
+    myOutputValue = drawMsg;
+    }
+      
+if ((playerSPS == 'scissors' && compSPS == 'paper') ||
+    (playerSPS == 'paper' && compSPS == 'stone') ||
+    (playerSPS == 'stone' && compSPS == 'scissors')){
+    myOutputValue = winMsg;
+    }
+
+if ((playerSPS == 'scissors' && compSPS == 'stone') ||
+    (playerSPS == 'paper' && compSPS == 'scissors') ||
+    (playerSPS == 'stone' && compSPS == 'paper')){
+    myOutputValue = loseMsg;
+    }
+return myOutputValue;
+}
+
+
+
+var randomSPS = function () {
+  randomInteger = Math.floor(Math.random() * 3);
+
+  if (randomInteger == 0){
+    sps = 'scissors';  
+  }
+  if (randomInteger == 1){
+    sps = 'paper';
+  }
+  if (randomInteger == 2){
+    sps = 'stone';
+  }
+  return sps;
+}
+
+
+
+// var loseRecord = 0;
+// var totalGame = 0;
+// var numOfGamePlayed = 0
+// var username = '';
+// var koreanWinner = '';
+// var koreanDraw = '';
+
+// var gameMode = 0;
+// var gameModeTypeMsg = 'Please select game mode type by entering the numeric number. <br><br> 1 - Normal Mode <br><br> 2 - Korean Mode <br><br> 3 - COM vs COM';
+
+// var main = function (input){
+//   if (!username){
+//     if (!input) {
+//       return 'Please enter your username.'
+//     }
+  
```

Comment:
When making a pull request, the convention is to remove comments from your code, unless they explain what the code is doing. Something to remember for your next project!